charts/iperf3/templates/deployment.yml

```diff
@@ -23,10 +23,12 @@ spec:
           command:
             - iperf3
             - -s
-            - -p 40000
+{{- if .Values.service.port }}
+            - -p {{.Values.service.port}}
+{{- end }}
           ports:
             - name: iperf3
-              containerPort: 40000
+              containerPort: {{.Values.service.port | default 5201}}
```

Comment:
this is the pod, not the service - there is not a lot of sense making the container port adjustable at all, since access should be going via the service.
So beside this is the wrong variable to take in the first place, i would argue we need a new one at all.

Reply:
I was trying to find out why the service was not reachable on one of my clusters, and therefore wanted to keep all used ports below the range assigned by Kubernetes for nodeports.

So I figured using one variable should be good enough. I daresay most people just want the default port.

But of course I can introduce a new variable. Hold on...